src/locale/languages/fr.js

```diff
@@ -92,6 +92,7 @@ module.exports = {
   'No contributors found': 'Aucun contributeur trouvé',
   'No description provided.': 'Aucune description fournie.',
   'No issues': 'Aucun ticket',
+  'No members found': '',
```

Comment:
Bonjour @machour! :smile: 

Can you please add a translation of this new phrase?

Reply:
`Aucun membre trouvé`? :D @lex111

Reply:
Perfect @olegkorol :)